README.md

```diff
@@ -64,6 +41,7 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
+- Original design idea by ##Cindy Shin## in ##Behance.##
```

Comment:
- [ ] Please make sure to add a link to Cindy's profile on Behance, so you can give the proper acknowledgement.